src/pages/Points.js

```diff
@@ -317,9 +317,9 @@ const FETCH_USER_QUERY = gql`
       fallPoints
       springPoints
       summerPoints
-      fallPercentile
-      springPercentile
-      summerPercentile
+      fallPercent
+      springPercent
+      summerPercent
```

Comment:
This change caused a little issue when I ran the code locally. You can and should change how the percentage is viewed in the frontend. However, when changing the variables we retrieve from the query call, our backend doesnt know what the new variables are and can't give us a correct response.